README.md

```diff
@@ -48,10 +48,21 @@ ccm uses python distutils so from the source directory run:
 
     sudo ./setup.py install
 
+ccm is available on the [Python Package Index][pip]:
+
+    pip install ccm
+
+There is also a [Homebrew package][brew] available:
+
+    brew install ccm
```

Comment:
It's at 2.0.3 right now. I can probably get a pull request together to bring it up to date.